srecord/arglex/tool/get_number.cc

```diff
@@ -98,20 +98,32 @@ srecord::arglex_tool::get_number(const char *caption)
     case token_round_down:
         token_next();
         multiple = get_number("-round-down");
+        if(multiple == 0)
```

Comment:
There should be a space after the `if`, and none after `fatal_error`

~~~C++
        if (multiple == 0)
        {
            fatal_error("-Round_Down value must not be 0");
~~~